manifest.md

```diff
@@ -85,13 +85,20 @@ A client will distinguish a manifest list from an image manifest based on the Co
 
     This OPTIONAL property contains arbitrary metadata for the manifest list.
     Annotations is a key-value, unordered hashmap.
-    Keys are unique, and best practice is to namespace the keys.
+    Keys MUST be unique, and best practice is to namespace the keys.
+	Keys starting with the `opencontainers.org` namespace are reserved and MUST NOT be used.
```

Comment:
On Mon, Jul 18, 2016 at 10:20:04AM -0700, Vincent Batts wrote:
> > -    Keys are unique, and best practice is to namespace the keys.
> > +    Keys MUST be unique, and best practice is to namespace the keys.
> > +	Keys starting with the `opencontainers.org` namespace are reserved and MUST NOT be used.
> 
> that ought to be clarified. I had in mind that it would be namespaced more
> like an URL, rather than a java package.

Does it matter?  And as long as folks don't have actual hosts like
net.example.com, folks could use both.  If we pick one ordering we
don't even have to worry about that, but I don't see any technical
difference between the two orderings.